src/terms/nonlocal.jl

```diff
@@ -77,10 +79,10 @@ end
                 forces[idx] += map(1:3) do α
                     dPdR = [-2T(π)*im*p[α] for p in G_plus_k] .* P
                     ψk = ψ[ik]
-                    dHψk = P * (C * (dPdR' * ψk))
+                    δHψk = P * (C * (dPdR' * ψk))
                     -sum(occupation[ik][iband] * basis.kweights[ik] *
-                         2real(dot(ψk[:, iband], dHψk[:, iband]))
-                         for iband=1:size(ψk, 2))
+                             2real(dot(ψk[:, iband], δHψk[:, iband]))
+                         for iband in axes(ψk, 2))
```

Comment:
We don't do this generally elsewhere in the code, we're fine with 1-based indexing